diff -urNad qrupdate-1.1.2-orig/Makeconf qrupdate-1.1.2/Makeconf
--- qrupdate-1.1.2-orig/Makeconf	2010-01-19 13:35:49.000000000 +0200
+++ qrupdate-1.1.2/Makeconf	2012-03-04 14:02:13.000000000 +0200
@@ -1,7 +1,7 @@
 # set this to your compiler's executable name (e.g. gfortran, g77)
 FC=gfortran
 # requested flags
-FFLAGS=-fimplicit-none -O3 -funroll-loops 
+FFLAGS=$(CFLAGS) -fimplicit-none -O3 -funroll-loops 
 # set if you need shared library
 FPICFLAGS=-fPIC
 
